eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -1313,3 +1471,152 @@ delayingActionsSpec =
           passNEpochs 2
           curConstitution <- getsNES $ newEpochStateGovStateL . constitutionGovStateL
           curConstitution `shouldBe` constitution
+
+committeeMaxTermLengthSpec ::
+  forall era.
+  ConwayEraImp era =>
+  SpecWith (ImpTestState era)
+committeeMaxTermLengthSpec =
+  describe "Committee members can serve full `CommitteeMaxTermLength`" $ do
+    let electMembers spoC drep = do
+          m1 <- KeyHashObj <$> freshKeyHash
+          m2 <- KeyHashObj <$> freshKeyHash
+          currentEpoch <- getsNES nesELL
+          maxTermLength <-
+            getsNES $
+              nesEsL . curPParamsEpochStateL . ppCommitteeMaxTermLengthL
+          let exceedingExpiry = addEpochInterval (addEpochInterval currentEpoch $ EpochInterval 1) maxTermLength
+              membersExceedingExpiry = [(m1, exceedingExpiry), (m2, exceedingExpiry)]
+          GovPurposeId gaid <-
+            electCommittee
+              SNothing
+              drep
+              Set.empty
+              membersExceedingExpiry
+          submitYesVote_ (StakePoolVoter spoC) gaid
+          pure (m1, m2)
+    it "maxTermLength = 0" $ do
+      -- ======== EPOCH e ========
+
+      let termLength = EpochInterval 0
+      modifyPParams $ \pp ->
+        pp
+          & ppCommitteeMaxTermLengthL .~ termLength
+
+      (drep, _, _) <- setupSingleDRep 1_000_000
+      (spoC, _, _) <- setupPoolWithStake $ Coin 10_000_000
+
+      -- Setup new committee members with max term length of 0 epoch
+      (c1, c2) <- electMembers spoC drep
+
+      curProtVer <- getProtVer
+      nextMajorVersion <- succVersion $ pvMajor curProtVer
+      let nextProtVer = curProtVer {pvMajor = nextMajorVersion}
+
+      -- Create a proposal for a hard-fork initiation
+      gid <- submitGovAction $ HardForkInitiation SNothing nextProtVer
+
+      -- Accept the proposal with all the governing bodies except for the CC
```

Comment:
I think it would be useful here to assert what the committee is - in order to clarify the premise of the test.